src/pages/index.tsx

```diff
@@ -6,87 +6,89 @@ import Button from "../components/commons/Button";
+          />
+        </div>
+        <div className="border-2 rounded-[25px] my-6 mx-auto w-[90%] sm:max-w-lg h-900px] z-30 bg-gray-100 p-3 flex flex-col">
+          <h1 className="rocket-title mb-8 text-center text-3xl font-bold tracking-tight text-gray-800">
+            Rocket
+          </h1>
+          <p className="rocket-description-1 mb-8 text-center text-xl font-bold tracking-tight text-gray-800">
+            The easiest way to connect with a wallet
+          </p>
+          <p className="rocket-description-2 mb-8 text-center text-xl font-bold tracking-tight text-gray-800">
+            Sign in methods
+          </p>
 
-        <div className="flex flex-col justify-center m-5 px-5 ">
-          {secret && (
-            <div className="flex flex-col justify-center px-5 ">
-              <Input
-                label="Secret key"
-                value={secret}
-                readOnly={true}
-                required={true}
-              />
-              <Input
-                label="Public key"
-                value={publicKey}
-                readOnly={true}
-                required={true}
-              />
-              <div className="flex flex-col justify-center m-5">
-                <Button buttonClass="button-confirm-wallet" secondary fullWidth>
-                  Confirm wallet
-                </Button>
-              </div>
-              <div className="flex flex-col justify-center m-5">
-                <Button
-                  buttonClass="button-go-back"
-                  onClick={() => {
-                    setPublicKey(null);
-                    setSecret(null);
-                  }}
-                  fullWidth
-                  danger
-                >
-                  Go back
-                </Button>
+          <div className="flex flex-col justify-center m-5 px-5 ">
+            {secret && (
+              <div className="flex flex-col justify-center px-5 ">
+                <Input
+                  label="Secret key"
+                  value={secret}
+                  readOnly={true}
+                  required={true}
+                />
+                <Input
+                  label="Public key"
+                  value={publicKey}
+                  readOnly={true}
+                  required={true}
+                />
+                <div className="flex flex-col justify-center m-5">
+                  <Button
+                    buttonClass="button-confirm-wallet"
+                    secondary
+                    fullWidth
+                  >
+                    Confirm wallet
+                  </Button>
+                </div>
+                <div className="flex flex-col justify-center m-5">
+                  <Button
+                    buttonClass="button-go-back"
+                    onClick={() => {
+                      setPublicKey(null);
+                      setSecret(null);
```

Comment:
We can save this in a function.

Reply:
Fix in commit e46bc72621e2e6b5b3167b81e5de57d91c8b23d3.